src/main/java/io/lettuce/core/api/async/RedisListAsyncCommands.java

```diff
@@ -35,56 +36,56 @@
     /**
      * Remove and get the first element in a list, or block until one is available.
      *
-     * @param timeout the timeout in seconds
-     * @param keys the keys
+     * @param timeout the timeout in seconds.
+     * @param keys the keys.
```

Comment:
Maybe it makes sense to regenerate API classes in a separate PR to avoid having this dots and commas changes in every single new PR.

Reply:
We don't need a new PR to regenerate the API with regards to comments update. Feel free to generate the API without the two commands, commit to `main` and rebase your PR onto of `main`. 

Probably we need to polish up the API generator to make sure that we don't accidentally delete methods. At least that was the case a few months ago.

Reply:
Ok, I'll check that out.

Reply:
Well, I've polished API over here https://github.com/lettuce-io/lettuce-core/commit/45e8307c6d4c9e2e6a5324454574d65510ca703c
And rebased this PR. Much better now :)

And yes, we still have to fix generator. I've seen such problems for now:
1) Dispatch methods (RedisSentinel) haven't generated. After API generating we still have to add explicitly. I propose to introduce separate template class with these methods.
2) imports! I haven't seen an opportunity to automatically clean up (remove useless, wrap as asterisk ones). However, there is an issue about it in javaparser (https://github.com/javaparser/javaparser/issues/1590), but I couldn't find this example or out-of-box API for our purpose.
3) Return type for `zmscore` generates incorrectly in kotlin API (wrong nullablity of generic list type)

I think we should raise on issue on that.

Reply:
Thanks a lot. To your points:

1. We should add `dispatch` methods to the `RedisSentinelCommands` template class (similar to `BaseRedisCommands`)
2. Imports are an issue, I also didn't find a good solution yet. However, reorganizing imports by the IDE (e.g. IntelliJ) before commit is fine for cleanup, so it wasn't a high priority issue yet.
3. Agree on `zmscore`. We had to do some trickery also for the reactive API (`Mono<List<Double>>`). 

Feel free to create issues for these topics.